presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -955,6 +963,28 @@ void VeloxQueryPlanConverterBase::parseIndexLookupCondition(
     return;
   }
 
+  if (const auto contains = isContains(filter)) {
+    VELOX_CHECK_EQ(contains->arguments.size(), 2);
+    const auto keyColumnExpr = exprConverter_.toVeloxExpr(
+        std::dynamic_pointer_cast<protocol::VariableReferenceExpression>(
+            contains->arguments[1]));
+    VELOX_CHECK_NOT_NULL(
+        keyColumnExpr, "{}", toJsonString(contains->arguments[1]));
+
+    const auto conditionColumnExpr = exprConverter_.toVeloxExpr(
+        std::dynamic_pointer_cast<protocol::VariableReferenceExpression>(
+            contains->arguments[0]));
+    VELOX_CHECK(
+        !core::TypedExprs::isConstant(conditionColumnExpr),
+        "The condition column needs to be not constant: {}",
+        toJsonString(filter));
+
+    joinConditionPtrs.push_back(
+        std::make_shared<core::InIndexLookupCondition>(
+            keyColumnExpr, conditionColumnExpr));
+    return;
+  }
+
```

Comment:
Do we have unit test for this?

Reply:
Current unit test won't be able to cover plan conversion.